src/message/http/index.ts

```diff
@@ -71,7 +71,7 @@ export function deserialize(message: Message): CloudEvent {
  * @returns {Mode} the transport mode
  */
 function getMode(headers: Headers): Mode {
-  const contentType = headers[CONSTANTS.HEADER_CONTENT_TYPE];
+  const contentType = headers[CONSTANTS.HEADER_CONTENT_TYPE] as string;
```

Comment:
i think you can add as const to the actual constants file instead of in every usage of this.
Ref: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions

in `sdk-javascript/src/constants.ts`
```js
const CONSTANTS = Object.freeze({
  CHARSET_DEFAULT: "utf-8",
  ... 
} } as const);

export default CONSTANTS;